predictor-server/src/main/java/ru/dbzukunft/predictor/server/entity/AlphaVantageTimeSeries.java

```diff
@@ -0,0 +1,23 @@
+package ru.dbzukunft.predictor.server.entity;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Builder
+@Getter
+public class AlphaVantageTimeSeries {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
+    @Column(unique = true)
```

Comment:
Currently there is no way to recognise what equity type this data belongs to. If you change type to something different and restart server - you'll get corrupted result. I would add "equityType" column and update unique constraint to "equityType+dateTime"

Reply:
AlphaVantageTimeSeries entity is special entity for AlphaVantage api. If we want to change type we should create new entity like "DifferentApiTimeSeries" for new api

Reply:
It still relates to AlphaVantage only, API returns time series for specified equity type, currently it is **Microsoft**:
`alphavantage.symbol=MSFT`
 however it can be changed. So following can happen:
1. Server started
2. Crawler hits
https://www.alphavantage.co/query?apikey=LA7KJ00H5WGTNMYQ&function=TIME_SERIES_INTRADAY&datatype=json&symbol=MSFT&outputsize=full&interval=5min
and saves **Microsoft** time series to DB
3. Somebody change type to **Apple**
`alphavantage.symbol= AAPL`
4. Server restarted
5. Crawler now hits
https://www.alphavantage.co/query?apikey=LA7KJ00H5WGTNMYQ&function=TIME_SERIES_INTRADAY&datatype=json&symbol=AAPL&outputsize=full&interval=5min
and saves **Apple** time series to same table 

=> you have mixed entries in Database